src/routes/dashboard/index.tsx

```diff
@@ -173,6 +173,7 @@ export function Dashboard() {
                     filter={{
                       startDate: dateFilter?.startDate,
                       endDate: dateFilter?.endDate,
+                      engagementRegions: selectedRegions,
```

Comment:
If this is a direct correlation to the API field name, I think it should be `engagementRegion` without the `s`.  If not ignore and I can approve.

Reply:
Good catch, it is translated in a different part of the application to be compatible with the api, so this should be good as is.